pepper-apis/src/main/java/org/broadinstitute/ddp/export/collectors/ActivityResponseCollector.java

```diff
@@ -222,6 +225,9 @@ private void collectQuestionMappings(Map<String, Object> props, Map<String, Obje
             case TEXT:
                 currProps.putAll(textFmt.mappings((TextQuestionDef) questionDef));
                 break;
+            case ACTIVITY_INSTANCE_SELECT:
```

Comment:
I don't see an OpenAPI spec update for this question type.

Reply:
Updated
<img width="982" alt="image" src="https://user-images.githubusercontent.com/71830706/140056640-0fdc095b-17dc-47a7-96cd-7ee9da5e1060.png">